wildlifelicensing/apps/applications/templates/wl/entry/upload_senior_card.html

```diff
@@ -0,0 +1,74 @@
+{% extends 'wl/base.html' %}
+
+{% load static %}
+
+{% load bootstrap3 %}
+
+{% load users %}
+
+{% block extra_css %}
+    <link href="{% static 'wl/css/application.css' %}" rel="stylesheet"/>
+{% endblock %}
+
+{% block breadcrumbs %}
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <ol class="breadcrumb wl-breadcrumbs">
+                    <li><a href="{% url 'home' %}">Home</a></li>
+                    {% if request.user|is_officer %}
+                        <li><a href="{% url 'wl_applications:create_select_customer' %}">Select or Create Customer</a>
+                        </li>
+                    {% endif %}
+                    <li><a href="{% url 'wl_applications:select_licence_type' %}">Select Licence Type</a></li>
+                    <li class="active">Upload Senior Card</li>
+                </ol>
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block content %}
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12">
+                {% with heading='Upload Senior Card' %}
+                    {% include 'wl/entry/header.html' %}
+                {% endwith %}
+                <p>
+                    With this type of licence you are eligible for a senior discount if you are the holder of a Senior
+                    Card.
+                    I you have a scanned or photographed image of it, please upload it here..
```

Comment:
First word should be "If", rather than I. Also no need for .. (just one full stop is fine).